pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-testcontainers</artifactId>
-    <version>3.1.0</version>
+    <version>3.0.0</version>
```

Comment:
So, versions `3.0.0` and `3.1.0` were not released before?

Reply:
Yes. I did just increase the version number and did not recognize that 3.0.0 was'n released